examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs

```diff
@@ -0,0 +1,192 @@
+            /// </summary>
+            [Option("customerId", Required = true, HelpText =
+                "The Google Ads customer ID for which the call is made.")]
+            public long CustomerId { get; set; }
+
+            /// <summary>
+            /// The start time of the seasonality adjustment (in yyyy-MM-dd HH:mm:ss format) in
+            /// the account's timezone.
+            /// </summary>
+            [Option("startDateTime", Required = true, HelpText =
+                "The start time of the seasonality adjustment (in yyyy-MM-dd HH:mm:ss format) " +
+                "in the account's timezone.")]
+            public string StartDateTime { get; set; }
+
+            /// <summary>
+            /// The end time of the seasonality adjustment (in yyyy-MM-dd HH:mm:ss format) in
+            /// the account's timezone.
+            /// </summary>
+            [Option("endDateTime", Required = true, HelpText =
+                "The end time of the seasonality adjustment (in yyyy-MM-dd HH:mm:ss format) in " +
+                "the account's timezone.")]
+            public string EndDateTime { get; set; }
+
+            /// <summary>
+            /// The conversion rate adjustment (an increase or a decrease), which accounts for
+            /// estimated changes in conversion rate due to a future event.
+            /// </summary>
+            [Option("conversionRateModifier", Required = true, HelpText =
+                "The conversion rate adjustment (an increase or a decrease), which accounts for " +
+                "estimated changes in conversion rate due to a future event.")]
+            public double ConversionRateModifier { get; set; }
+        }
+
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            Options options = new Options();
+            CommandLine.Parser.Default.ParseArguments<Options>(args).MapResult(
+                delegate (Options o)
+                {
+                    options = o;
+                    return 0;
+                }, delegate (IEnumerable<Error> errors)
+                {
+                    // The Google Ads customer ID for which the call is made.
+                    options.CustomerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+                    // The start time of the seasonality adjustment (in
+                    // yyyy-MM-dd HH:mm:ss format) in the account's timezone.")]
+                    options.StartDateTime = "INSERT_START_DATE_TIME_HERE";
+
+                    // The end time of the seasonality adjustment (in
+                    // yyyy-MM-dd HH:mm:ss format) in the account's timezone.")]
+                    options.EndDateTime = "INSERT_END_DATE_TIME_HERE";
+
+                    options.ConversionRateModifier =
```

Comment:
Missing explanatory comment for this argument.

Reply:
Done.